Source/WPEFramework/Config.h

```diff
@@ -488,7 +593,7 @@ namespace PluginHost {
 
         public:
             InputInfo(const InputInfo&) = delete;
-            InputInfo& operator= (const InputInfo&) = delete;
+            InputInfo& operator=(const InputInfo&) = delete;
```

Comment:
If we delete the assignment operator for const <Object>&) we should also specifically delete the assignment operator  with the move parameter(<Object>&&).

Reply:
As we discussed need to cross check

Reply:
done